static/CaseTester.css

```diff
@@ -26,4 +26,6 @@ td{
 th td {
 	display: table-cell;
 	vertical-align: inherit;
-}
\ No newline at end of file
+}
+a:hover {
+	cursor: default}
```

Comment:
i recommend changing this to cursor: pointer; so that it functions like an actual link